src/components/Dropzone.svelte

```diff
@@ -32,6 +32,8 @@
   export let containerStyles = "";
   export let disableDefaultStyles = false;
   export let name = "";
+  export let inputElement;

```

Comment:
Thanks for changing its name, but notice you need to change all of the other references of `inputRef` in the code, to become `inputElement`.
To be sure, I suggest to check if it gets compiled well after renaming.

Reply:
hehe yea that went a bit quick